agent.go

```diff
@@ -659,13 +673,6 @@ func (a *Agent) AddRemoteCandidate(c Candidate) error {
 		return nil
 	}
 
-	// TCP Candidates with TCP type active will probe server passive ones, so
```

Comment:
I moved this lower. I needed this function to get into `addPair` so I could thread in a call to `CreatePermissions`. I now have `addPair` check this and bail. This avoids starting up connectivity checks where the other side is making the initial connection.